zebra-state/src/service/finalized_state.rs

```diff
@@ -373,12 +373,30 @@ impl FinalizedState {
             let block_time = block.header.time.timestamp();
             let local_time = chrono::Utc::now().timestamp();
 
-            const AWAY_FROM_TIP_BULK_SIZE: usize = 800;
+            // Mainnet bulk size is small enough to avoid the elasticsearch 100mb content
+            // length limitation. MAX_BLOCK_BYTES = 2M. Each block count as 2 as we send
+            // them with a operation/header line. A value of 100 is 50 blocks.
+            const MAINNET_AWAY_FROM_TIP_BULK_SIZE: usize = 100;
```

Comment:
Block data is hex-encoded, so a 2MB binary block actually creates 4MB of JSON. Some block and transaction data is repeated multiple times in JSON, so I'd suggest allowing a little bit more space as well.

```suggestion
            const MAINNET_AWAY_FROM_TIP_BULK_SIZE: usize = 45;
```

Reply:
I will like to test as it is and if it not work maybe suggest the users to increase ` http.max_content_length` in elasticsearch to `200` or try compressing the data. 

Doing such small blocks will impact the performance of the sync (i am not really sure if the current change will make the sync in lets say 1 week, i can't imagine doing it with bulks of 20 blocks).

By the way, using an even number there will break the program as each block is 2 lines.

Reply:
Maybe i will add this value to the config so users can test and make changes without changing the code. If zebra panics and the user change the config value it can restart pretty much where it was, elastic will just replace repeated entries.